source/_posts/2021-03-06-android-2021-q1-releases.markdown

```diff
@@ -0,0 +1,93 @@
+
+A new notification command was added to allow the user to launch an activity on their android device. This command requires a new permission to be granted in order to launch activities from the background, Draw Over Other Apps. The first attempt to use this notification will take the user to the permission page so the user can grant proper access. It is important to note that if the app is not considered active then this permission page will not show up due to missing permissions. Try to test this with the app open or you can grant the permission manually in your device settings. There are lots of use cases for this feature such as being able to launch Google Maps driving mode or even setting an alarm on your device.
+
+Unfortunately, it is not so straightforward to determine what intents and activities are supported by applications. You really need to know what to look for and there is not much in terms of documentation here from applications. Try reaching out to your favorite apps developers to see if they have any intents to consume. We have provided several live examples in the [companion docs](https://companion.home-assistant.io/docs/notifications/notification-commands#activity). I have also started a new thread in the forums to maintain a list of all that we can find [here](https://community.home-assistant.io/t/android-intents-sending-receiving-list-starting-activities-too/276192) I will be trying to keep the first post as up to date as possible.
+
+## Other Enhancements
+
+We have also spent time making improvements to all other areas too:
+
+- Support for links from [My Home Assistant](https://my.home-assistant.io)
+- Power menu fixes and enhancements including support for vacuum entities
+- Haptic feedback support in the frontend
+- Overriding certain URL types to launch an application or an intent from the frontend
+- 3 finger swipe down gesture to trigger the Quick Bar
+
+
+Big thank you to everyone involved. Please keep those bug reports and feature requests coming!
+
+## Changelogs
+
+- 2021.1 - https://github.com/home-assistant/android/releases/tag/2021.1.1
+- 2021.2 - https://github.com/home-assistant/android/releases/tag/2021.2.1
+- 2021.3 - https://github.com/home-assistant/android/releases/tag/2021.3.1
```

Comment:
There were a couple minor ones in between, not sure if they are worth a mention or not.

Reply:
Ah good catch, the version numbers are off too.  Will update and change the date.